src/Compilers/CSharp/Portable/CSharpResources.Designer.cs

```diff
@@ -5831,6 +5831,15 @@ internal class CSharpResources {
         }
         
         /// <summary>
+        ///   Looks up a localized string similar to &apos;{0}&apos; cannot implement interface member &apos;{1}&apos; in type &apos;{2}&apos; because feature &apos;{3}&apos; is not available in C# {4}. Please use language version {5} or greater..
+        /// </summary>
+        internal static string ERR_LanguageVersionDoesNotSupportDefaultInterfaceImplementationForMember {
```

Comment:
Nevermind, I see that you already have a PROTOTYPE marker ;-)
Thanks!

---
In reply to: [119430273](https://github.com/dotnet/roslyn/pull/19876#discussion_r119430273) [](ancestors = 119430273)